Tribler/Core/Libtorrent/LibtorrentDownloadImpl.py

```diff
@@ -1173,10 +1181,6 @@ def dlconfig_changed_callback(self, section, name, new_value, old_value):
     def get_share_mode(self):
         return self.handle.status().share_mode
 
-    @waitForHandleAndSynchronize(True)
-    def set_share_mode(self, share_mode):
```

Comment:
Yes, I thought about putting it back. The use of a flag on_add is working as far as the assert can guarantee.

But I am wondering if this method is stable enough to use after the torrent has been added.
@ardhipoetra Is the one to find that one out.